service/src/main/java/bio/terra/policy/db/PaoDao.java

```diff
@@ -74,6 +77,42 @@ public PaoDao(TpsDatabaseConfiguration tpsDatabaseConfiguration) {
     this.tpsJdbcTemplate = new NamedParameterJdbcTemplate(tpsDatabaseConfiguration.getDataSource());
   }
 
+  @Retryable(interceptor = "transactionRetryInterceptor")
+  @Transactional(
+      isolation = Isolation.SERIALIZABLE,
+      propagation = Propagation.REQUIRED,
+      transactionManager = "tpsTransactionManager")
+  public void clonePao(UUID sourceObjectId, UUID destinationObjectId) {
+    DbPao sourcePao = getDbPao(sourceObjectId);
+
+    final String sql =
```

Comment:
Maybe we can factor out the insert statement and share it with create?

nit: maybe use """ form for the query strings instead of concatenating.